html2asketch/nodeTreeToSketchGroup.js

```diff
@@ -0,0 +1,58 @@
+import nodeToSketchLayers from './nodeToSketchLayers';
+import Group from './group';
+import Style from './style';
+
+function isNotEmpty(value) {
+  return value !== undefined && value !== null && value !== '';
+}
+
+export default function nodeTreeToSketchGroup(node, options) {
+  const bcr = node.getBoundingClientRect();
+  const {left, top} = bcr;
+  const width = bcr.right - bcr.left;
+  const height = bcr.bottom - bcr.top;
+
+  // Collect layers for the node level itself
+  const layers = nodeToSketchLayers(node, options) || [];
+
```

Comment:
let's check here if node we are trying to process is visible to avoid empty groups